### PR TITLE
Our game is blooming (onion)

### PR DESCRIPTION
**Pull request by Matthew-Stewart**
This is good to go, still need to work on the frame rate a bit though (if anyone wants to help me with optimizations let me know)

### CODE REVIEW

File: src/game_renderer/GameRenderer.cpp [hunk 1]
```diff
@@ -32,8 +32,28 @@
...
+GLuint hdrFBO;
+GLuint hdrColorBuffers[2];
+GLuint pingpongFBO[2];
+GLuint pingpongColorbuffers[2];*/
+/*std::unordered_map<std::string,
+                   std::shared_ptr<Program>> GameRenderer::programs;
+GLuint GameRenderer::hdrFBO;
+GLuint GameRenderer::hdrColorBuffers[2];
+GLuint GameRenderer::pingpongFBO[2];
+GLuint GameRenderer::pingpongColorbuffers[2];*/
```
Comment (bbeck13) [inline src/game_renderer/GameRenderer.cpp]: Can you delete this commented out stuff

**Review by bbeck13 — COMMENTED** [review]
looks good 1 minor comment QA 🐡

**Review by josepharhar — COMMENTED** [review]
CR 🌱 QA 🌷

**Comment by alexottoboni** [comment]
So I had a suspicion that the display scaling is part of why the frame-rate gets hit so hard since the GPU has to write the entire frame to a temporary texture and then write it again to screen. So a high res display with an integrated graphics card doesn't have enough memory for that to be fast.
Changing my display resolution from 1680x1050 to 1024x8XX doubles my framerate.

Since culling and all that optimization stuff doesn't help display size, maybe we can demo on reduced screen resolution?

I'll look for other ways to speed up the post processsing.

**Comment by Matthew-Stewart** [comment]
I was thinking of just saying if screen resolution is over some width x height, divide the width and height by 2.
Also note that the problem is even worse on retina displays (for example mine is 2880x1800, even though i only have 1440x900 actual pixels)

**Comment by bbeck13** [comment]
If its an issue we can demo on my laptop its 1366X768 and seems to run the bloom fine

**Comment by bbeck13** [comment]
CR 🇯🇲  QA 🍁

**Pull request merged**